back/engines/free/email_campaigns/spec/mailers/previews/email_campaigns/moderator_digest_mailer_preview.rb

```diff
@@ -19,27 +19,11 @@ def campaign_mail
           project_name: project_name,
           statistics: {
             activities: {
```

Comment:
Given we are reducing these objects massively, do we even need `activities` or `users` as groupings?

Reply:
Nope, and I agree. I'll just move them up to all be in `statistics:`.